tls_examples/AGS_AerospikeDB/aerospike.conf

```diff
@@ -0,0 +1,50 @@
+# Aerospike database configuration file.
+
+service {
+    feature-key-file /etc/aerospike/features.conf
+    cluster-name exampleCluster # Specify cluster name for tls to check
+}
+
+logging {
+    console {
+        context any info
+        context tls debug # Enable debug logging for TLS
+    }
+}
+
+network {
+    tls exampleCluster { # TLS context with name "exampleCluster"
+        ca-file /opt/aerospike/etc/ca.crt # CA certificate to verify certs
+        cert-file /opt/aerospike/etc/server.crt # Server certificate shown to clients/peers
+        key-file /opt/aerospike/etc/server.key # Private key for server cert
+    }
+
+    service {
+        port 3000
+        tls-address any # listen to all interfaces for TLS connections
+        tls-port 4000 # TLS port for client/API connections
+        tls-name exampleCluster # Match this name with the TLS context block name
+        tls-authenticate-client false
+    }
+
+    heartbeat {
+            mode mesh
```

Comment:
Please align spacing across this file, the default is 4 spaces but in `heartbeat` stanza its 8 and in `namespace` stanza its 2